server/src/server.ts

```diff
@@ -247,20 +250,25 @@ async function validateTextDocument(textDocument: TextDocument): Promise<void> {
 		connection.console.log(`Temp source file ${reachTempIndexFile} saved!`);
 	});
 
-	await exec("cd " + tempFolder + " && " + path.join(process.cwd(), "reach") + " compile " + REACH_TEMP_FILE_NAME, (error: { message: any; }, stdout: any, stderr: any) => {
+	await exec("cd " + tempFolder + " && reach " + path.join(process.cwd() , "reach") + " compile " + REACH_TEMP_FILE_NAME + " --error-format-json", (error: { message: any; }, stdout: any, stderr: any) => {
```

Comment:
This doesn't need `reach` before `path.join(process.cwd() , "reach")` since the latter is the absolute path to the `reach` script in the project directory.  This command is run from the temp directory (where the temporary copy of the working document is saved) so it does not have a `reach` script.

Reply:
Oops, thanks for catching. That was leftover from debugging. I'll revert that.